dask/bytes/s3.py

```diff
@@ -62,5 +86,10 @@ def _get_pyarrow_filesystem(self):
         import pyarrow as pa
         return pa.filesystem.S3FSWrapper(self)
 
+    def commit(self, metadata):
+        self._call_s3(
+            self.s3.complete_multipart_upload,
+            **metadata)
```

Comment:
Interesting, I didn't know that this existed.  Does this often exist in other systems as well?  HDFS, GCS, etc..

Reply:
The model for HDFS is typically to write things to a subdirectory and move them to their final location.  I would hope that most of the other blob stores support something similar.